external/src/main/java/com/redhat/cloud/policies/engine/handlers/TriggersHandler.java

```diff
@@ -657,14 +658,15 @@ public void deleteGroupTrigger(RoutingContext routing) {
             @DocResponse(code = 404, message = "Trigger not found", response = ResponseUtil.ApiError.class),
             @DocResponse(code = 500, message = "Internal server error.", response = ResponseUtil.ApiError.class)
     })
-    @Timed(absolute = true, tags = {"path=/triggers/{triggerId}", "method=DELETE"})
+    @Timed
+            //(absolute = true, tags = {"path=/triggers/{triggerId}", "method=DELETE"})
```

Comment:
Are the old tags automatically added by Micrometer? Should that comment be removed?

Reply:
I tried to somehow add the tags with the last cmmit, but absolute = true isn't available